File: multibanking-app/src/api/Bank.ts
import { BankLoginSettings } from "./BankLoginSettings";
import { BankApi } from "./BankApi";

export interface Bank {
    "bankApi"?: BankApi;
    "bankCode"?: string;
    "bic"?: string;
    "blzHbci"?: string;
    "id"?: string;
    "loginSettings"?: BankLoginSettings;
    "name"?: string;
    "searchIndex"?: Array<string>;
}

File: onlinebanking-facade/src/main/java/domain/Bank.java
package domain;

import lombok.Data;

/**
 * Created by alexg on 10.07.17.
 */
@Data
public class Bank {

    private String bankingUrl;
    private String bankCode;
    private String bic;
    private String name;
    private BankLoginSettings loginSettings;
    private BankApi bankApi;

}
